doc/examples/features_detection/plot_gabors_from_astronaut.py

```diff
@@ -51,7 +51,7 @@
 patch_shape = 8, 8
 n_filters = 49
 
-astro = color.rgb2gray(data.astronaut())
+astro = np.ascontiguousarray(color.rgb2gray(data.astronaut()))
```

Comment:
As this an _example_ I'd suggest to add a comment why `np.ascontiguousarray` is used.